src/canvas/StaticCanvas.ts

```diff
@@ -417,20 +417,6 @@ export class StaticCanvas<
     return (this._offset = getElementOffset(this.lowerCanvasEl));
   }
 
-  /**
-   * @private
-   */
-  protected _createCanvasElement() {
-    const element = createCanvasElement();
-    if (!element) {
-      throw new Error(CANVAS_INIT_ERROR);
-    }
-    if (typeof element.getContext === 'undefined') {
-      throw new Error(CANVAS_INIT_ERROR);
-    }
-    return element;
-  }
```

Comment:
This will error later with a different less clear error
something like `can't use <insertFunctionName> on null` or similar.

Do we want to remove this wrapper and why?

Reply:
I will make it part of createCanvasElement util

Reply:
6e48007d2